database/aws-s3.js

```diff
@@ -0,0 +1,109 @@
+const { config } = require('../config.js');
+const { S3Client, ListBucketsCommand, CreateBucketCommand, PutObjectCommand, DeleteBucketCommand } = require('@aws-sdk/client-s3');
+const { aws: { AWS_REGION, AWS_BUCKET_NAME, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY } } = config;
+const AWS = require('aws-sdk');
+
+const s3v2 = new AWS.S3();
+
+const s3 = new S3Client({ region: process.env.AWS_REGION });
+
+AWS.config.update({
+  accessKeyId: AWS_ACCESS_KEY_ID,
+  secretAccessKey: AWS_SECRET_ACCESS_KEY
+});
+
+// Get all buckets
+const getBuckets = async () => {
+  try {
+    const bucketList = await s3.send(new ListBucketsCommand({}));
+    console.log(bucketList.Buckets);
+  } catch(err) {
+    console.log(err);
+  }
+};
+
+// creates a new bucket
+
+const createBucket = async () => {

```

Comment:
Might be good to note in the readme that the bucket shouldn't already be created in S3. With Sam's, I think I created the bucket and then supplied the name and the script just populated it